composer.json

```diff
@@ -23,13 +24,19 @@
             "T3Docs\\Typo3DocsTheme\\": "packages/typo3-docs-theme/src/"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "T3Docs\\Typo3DocsTheme\\": "tests/"
```

Comment:
Why tests folder not under "packages/typo3-docs-theme/tests/". I assume, there may be some other packages in the future? That's why we have a monorepo?

Reply:
This is an integration test that does not only test our theme but the complete integration with other installed packages etc

Reply:
Ack